hashintel/hash
# H-199: Add Linear connection ontology types

## 🌟 What is the purpose of this PR?




This PR adds an `@linear` HASH user, with the `User`, `Organization` and `Issue` ontology types.

## 🔗 Related links





- ...

## Pre-Merge Checklist 🚀

### 🚢 Has this modified a publishable library?




This PR:


### 📜 Does this require a change to the docs?




The changes in this PR:


### 🕸️ Does this require a change to the Turbo Graph?




The changes in this PR:


## ⚠️ Known issues



## 🐾 Next steps



## ❓ How to test this?



1. Set the `LINEAR_CLIENT_ID` environment variable
2. Run the local developer environment, which should seed the types in the database
